src/sentry/search/eap/uptime_check_columns.py

```diff
@@ -0,0 +1,93 @@
+from sentry_protos.snuba.v1.request_common_pb2 import TraceItemType
+
+from sentry.search.eap import constants
+from sentry.search.eap.columns import (
+    ColumnDefinitions,
+    ResolvedColumn,
+    VirtualColumnDefinition,
+    datetime_processor,
+    project_context_constructor,
+    project_term_resolver,
+)
+from sentry.search.eap.common_columns import COMMON_COLUMNS
+
+UPTIME_CHECK_ATTRIBUTE_DEFINITIONS = {
+    column.public_alias: column
+    for column in COMMON_COLUMNS
+    + [
+        ResolvedColumn(
+            public_alias="environment",
+            internal_name="sentry.environment",
+            search_type="string",
+        ),
+        ResolvedColumn(
+            public_alias="uptime_subscription_id",
+            internal_name="uptime_subscription_id",
+            search_type="string",
+        ),
+        ResolvedColumn(
+            public_alias="uptime_check_id",
+            internal_name="uptime_check_id",
+            search_type="string",
+        ),
+        ResolvedColumn(
+            public_alias="scheduled_check_time",
+            internal_name="scheduled_check_time",
+            search_type="string",
+            processor=datetime_processor,
+        ),
+        ResolvedColumn(
+            public_alias="timestamp",
+            internal_name="timestamp",
+            search_type="string",
+            processor=datetime_processor,
+        ),
+        ResolvedColumn(
+            public_alias="duration_ms",
+            internal_name="duration_ms",
+            search_type="number",
+        ),
+        ResolvedColumn(
+            public_alias="region",
+            internal_name="region",
+            search_type="string",
+        ),
+        ResolvedColumn(
+            public_alias="check_status",
+            internal_name="check_status",
+            search_type="string",
+        ),
+        ResolvedColumn(
+            public_alias="check_status_reason",
+            internal_name="check_status_reason",
+            search_type="string",
+        ),
+        ResolvedColumn(
+            public_alias="http_status_code",
+            internal_name="http_status_code",
+            search_type="number",
+        ),
+        ResolvedColumn(
+            public_alias="trace_id",
+            internal_name="trace_id",
+            search_type="string",
+        ),
```

Comment:
https://github.com/getsentry/sentry/blob/8279a70bfe9666ea51f8685032efc2f17ef22969/src/sentry/search/eap/span_columns.py#L104-L109

Are these the same things? Should the be named the same?

Reply:
we don't use the `sentry.~` prefixes in uptime's schema (see above)